src/mqt/predictor/rl/PredictorEnv.py

```diff
@@ -181,8 +184,15 @@ def apply_action(self, action_index):
                 try:
                     altered_qc = pm.run(self.state)
                 except Exception as e:
-                    print("Error in executing Qiskit transpile pass: ", action_index)
-                    print(e)
+                    self.logger.error(
+                        "Error in executing Qiskit transpile pass: "
+                        + ", "
+                        + action["name"]
+                        + ", "
+                        + self.state.name
+                        + ", "
+                        + e
+                    )
```

Comment:
should probably be an exception, right? not really recoverable... or do you just continue and count this pass as a "failure"?

Reply:
So far I counted it as failure and continued. Do you think this is inappropriate?

Reply:
Also true. I added that.